IHP/IDE/SchemaDesigner/Parser.hs

```
@@ -229,3 +266,6 @@ comment = do
     lexeme "--" <?> "Line comment"
     content <- takeWhileP Nothing (/= '\n')
     pure Comment { content }
+
+tryTextToInt :: Text -> Int
+tryTextToInt input = fromMaybe 0 (textToInt input)
```

This should not silently fail